### PR TITLE
{devel}[intel/2018.00] SWIG v3.0.12

### PR DESCRIPTION
**Pull request by reedts**
(created using `eb --new-pr`)
depends on ~~#5129~~, ~~#5259~~

### CODE REVIEW

**Comment by boegel** [comment]
@reedts Please remove `SWIG-3.0.12-intel-2017.09-Python-3.6.2.eb` from this PR?

File: easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-intel-2018.00-Python-3.6.2.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'intel', 'version': '2018.00'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-intel-2018.00-Python-3.6.2.eb]: @reedts Please include a SHA256 checksum.

Comment (boegel) [reply easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-intel-2018.00-Python-3.6.2.eb]: @reedts Still missing SHA256 checksum, please add it?

Comment (reedts) [reply easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-intel-2018.00-Python-3.6.2.eb]: @boegel My bad, pushed to early. Here you go!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2609.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7026e87d8c63288fbef33ed4c4b68dcd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2c76a2b01cb8b84989a63e832b8dfa6f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @reedts!

**Pull request merged**